### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by omarismail94**
Add me to CodeOwners

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -2,5 +2,5 @@
 * @aditya-07 @deepankarb @jingtang10 @kevinmost @ktarasenko @omarismail94 @stevenckngaa @vvikramraj @yigit @wantingzhang
```
Comment (jingtang10) [inline CODEOWNERS:2]: can you also correct the order of these names? yigit and wantingzhang are the other way around.

**Review by jingtang10 — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (omarismail94) [reply CODEOWNERS]: fixed!

File: CODEOWNERS [hunk 2]
```diff
@@ -1,6 +1,5 @@
 # Owners of the repository
-* @aditya-07 @deepankarb @jingtang10 @kevinmost @ktarasenko @omarismail94 @stevenckngaa @vvikramraj @yigit @wantingzhang
-
+* @aditya-07 @deepankarb @jingtang10 @kevinmost @ktarasenko @omarismail94 @stevenckngaa @vvikramraj @wantingzhang @yigit
```
Comment (jingtang10) [inline CODEOWNERS:2]: actually can you update @wantingzhang to @wantingzhang77?
it was a mistake while adding it

**Review by jingtang10 — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 2 above)
Comment (omarismail94) [reply CODEOWNERS]: updated

**Review by jingtang10 — APPROVED** [review]

**Pull request merged**